tests/integration/test_zfs_generic.py

```diff
@@ -29,6 +29,10 @@
     # arc
     "arc",
 
+    # blkptr
+    "spa | head 1 | deref |member spa_uberblock | member ub_rootbp | blkptr",
+    "dbuf | head 1 | member db_blkptr | blkptr",
```

Comment:
adding these tests is great.  Don't you need to also add the expected output files?